src/index.scss

```diff
@@ -20,6 +20,10 @@ pre {
   font-weight: unset;
 }
 
+input {
+  @extend .fontBase;
+}
```

Comment:
It looks like this was missing as we don't specify a font-family for our inputs anywhere